bin/medic.js

```diff
@@ -39,9 +65,109 @@ var argv = optimist
     .argv;
 
 // helpers
+function killTasks(taskNames) {
+
+    if (!taskNames || taskNames.length < 1) {
+        console.warn("no tasks to kill");
+        return;
+    }
+
+    if (isWindows()) {
+        var cli  = "taskkill /F";
+        var args = taskNames.map(function (name) { return "/IM \"" + name + "\""; });
+    } else {
+        var cli  = "killall";
+        var args = taskNames.map(function (name) { return "\"" + name + "\""; });
+    }
+
+    var command = cli + " " + args.join(" ");
+    medicLog("running the following command:");
+    medicLog("    " + command);
+
+    shelljs.exec(command, {silent: false, async: true}, function (returnCode, output) {
+        if (returnCode !== 0) {
+            console.warn("WARNING: kill command returned " + returnCode);
+        }
+    });
+}
+
 function fatal(message) {
-    console.error("ERROR: " + message);
-    process.exit(1);
+    console.error("FATAL: " + message);
+    process.exit(ERROR);
+}
+
+function medicLog(message) {
+    console.log("\033[32m[MEDIC LOG]\033[m " + message);
```

Comment:
Might be worth adding a comment so that people can parse out what your escaped characters are.  They look like octal encodings instead of Unicode.  While valid, is uncommon because translating to octal and then to Latin-1 is a two-step process.